### PR TITLE
A short explanation

### PR DESCRIPTION
**Pull request by pmk01**

### CODE REVIEW

File: wcivf/apps/parties/templates/parties/party_list.html [hunk 1]
```diff
@@ -12,7 +12,10 @@
 <section class="columns large-12">
   <div class="card">
     <div class="row">
-      <h2>UK Political parties</h2>
+      <h2>UK political parties</h2>
+      The following is a list of all political parties represented by candidates in Democracy Club's UK election database (est. 2015). Party names and emblems are taken from the [Electoral Commission’s](http://www.electoralcommission.org.uk/) register of political parties.
```
Comment (symroe) [inline wcivf/apps/parties/templates/parties/party_list.html:16]: For both of these lines, could you wrap them in a `<p>` tag (that is `<p>[words]</p>` for each line?

This isn't needed if the text is in a markdown block, but is needed if it's just HTML (e.g, not in a `{% markdown %}` block)

Comment (symroe) [reply wcivf/apps/parties/templates/parties/party_list.html:16]: (Let me know if the GitHub UI makes this hard and I can do it)

Comment (pmk01) [reply wcivf/apps/parties/templates/parties/party_list.html]: Thanks, I think I've done this? I assumed this needed to be a commit, not a PR...

File: wcivf/apps/parties/templates/parties/party_list.html [hunk 2]
```diff
@@ -12,7 +12,11 @@
 <section class="columns large-12">
   <div class="card">
     <div class="row">
-      <h2>UK Political parties</h2>
+      <h2>UK political parties</h2>
+      <p>The following is a list of all political parties represented by candidates in Democracy Club's UK election database (est. 2015). The list includes active, deregistered, and defunct parties.</p>
+        
+       <p>Party names and emblems are taken from the [Electoral Commission’s](http://www.electoralcommission.org.uk/) register of political parties.<p/>
```
Comment (symroe) [inline wcivf/apps/parties/templates/parties/party_list.html:18]: So close on this one - you want `</p>` not `<p/>`

**Comment by symroe** [comment]
Nice one!

**Pull request merged**